routes/permafrost.py

```diff
@@ -296,6 +361,26 @@ def run_huc_fetch_all_permafrost(huc_id):
     magt_huc_pkg = package_gipl_polygon(magt_poly_sum_di)
     alt_huc_pkg = package_gipl_polygon(alt_poly_sum_di)
     combined_pkg = combine_gipl_poly_var_pkgs(magt_huc_pkg, alt_huc_pkg)
+
```

Comment:
Maybe use a `create_csv` function like what was used in `routes/alfresco.py` to do some deduplication? At least within this module and any others for consistency, but could potentially be imported `create_csv` function just can't tell from a quick look how generalizable it is.

Reply:
I wasn't able to get this done before the PR was merged, but I've created issue #147.